packages/fxa-auth-server/lib/senders/emails/partials/index.ts

```diff
@@ -0,0 +1,2 @@
+export * as cadReminder from './appBadges';
+export * as welcome from './button';
```

Comment:
I'm confused by this export name `welcome`, could you elaborate?

Reply:
Ahh yes I messed up while rebasing the branch. I'll fix it and update you soon